tests/Feature/Subscription.php

```diff
@@ -799,4 +799,48 @@ public function testSubscriptionSearchWithDaysPastDue()
         $overdue = $query->whereDaysPastDue('=', -2)->fetch();
         $this->assertEquals(0, count($overdue));
     }
+
+    /**
+     * @group integration
+     * @dataProvider customerProvider
+     */
+    public function testSubscriptionUpgradeCrossPlanWithUpdateToDefaultAddOns(callable $customerFactory)
+    {
+        $customer = $customerFactory($this->dues);
+
+        // Create initial subscription
+        $plan = $this->dues->findPlanById('401m');
```

Comment:
I wonder if we should document how these plans are set up. A ton of our tests probably run under a ton of assumptions about plans. Not needed now, just thinking out loud.